fontmaker/core/ttfparser.go

```diff
@@ -204,65 +204,69 @@ func (t *TTFParser) SetUseKerning(use bool) {
 }
 
 //Parse parse
-func (t *TTFParser) Parse(filepath string) error {
+func (t *TTFParser) Parse(filepath string) ([]byte, error) {
```

Comment:
I was very confused with this function. I thought the parser loads a font file and produce some *parsed* bytes and returns it to a caller but it wasn't. This function just returns a loaded font bytes as-is.

I think the original design still makes sense to me. `TTFParser` should  hold a loaded font bytes as is in the private struct field and give an access via `FontData()`.

Whenever `Parse()` or `ParseByReader()` are called, the parser should update their internal state with a loaded font bytes. I don't think we need to return from here.

Reply:
It was more for font data verification rather than parsing font data. `parse()` does [some sort of verification](https://github.com/HENNGE/gopdf/blob/experiment/fontmaker/core/ttfparser.go#L239-L325) so that we can ensure whether given font data is valid or not if we went through this function before getting font data.
Since [gopdf can have multiple `SubsetFontObj`](https://github.com/HENNGE/gopdf/blob/experiment/gopdf.go#L444-L457), I also wanted to directly get a specific font data after parsing font data (a.k.a. font data verification).

But I think your point still makes sense. Maybe, I can add a function that returns a set of font data. The interface would be:
1. Add several fonts
2. Get a set of loaded fonts at once
3. Set loaded fonts to another gopdf instance

Reply:
( We don't need to get loaded font data from gopdf instance. We can simply add font data as it is. 🙊 )